core/network/src/peer/mod.rs

```diff
@@ -64,7 +64,6 @@ mod test {
 
         let join_handle = tokio::task::spawn(async move {
             let server_config = ServerConfig::builder()
-                .with_safe_defaults()
```

Comment:
https://docs.rs/rustls/latest/src/rustls/server/server_conn.rs.html#349-354

Looks like "safe defaults" are automatically applied to the builder